routes/users/login.js

```diff
@@ -0,0 +1,26 @@
+const { MongoClient } = require("mongodb")
+const md5 = require("md5")
+const jwt = require("jsonwebtoken")
+
+const { URI, DB_NAME, tokenKey } = require("../../config/setup")
+const { ErrRes, SuccRes } = require("../../utils/helpers")
+
+const login = (req, res) => {
+  const { email, password } = req.body
+  const pw = md5(password)
+  MongoClient.connect(URI, { useNewUrlParser: true }, (err, db) => {
+    if (err) throw err
+
+    db.db(DB_NAME)
+      .collection("korisnici")
+      .findOne({ email, pass: pw }, (err, user) => {
+        if (!user) {
+          return res.status(400).send(new ErrRes("Pogresan email ili password"))
+        }
+        const token = jwt.sign({ user }, tokenKey)
```

Comment:
do sada mi se sve cini dobro, pravis token za korisnika i saljes mu, ali cini mi se da ga nigde ne cuvas u bazi, pa ga ne mozes prepoznati po tokenu kada se vrati... dobro, kontam da je to sledeci korak